docker/ol-solr-updater-start.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+python --version
+python scripts/new-solr-updater.py \
```

Comment:
Remove this section from the home start script: https://github.com/internetarchive/openlibrary/blob/master/docker/ol-home-start.sh#L30 ; it's no longer needed there :)